src/sql/src/session/vars/definitions.rs

```diff
@@ -1801,31 +1801,31 @@ feature_flags!(
         enable_for_item_parsing: true,
     },
     {
-        name: enable_table_check_constraint,
```

Comment:
I don't think they are, but do we currently have any of these in LaunchDarkly? If so changing the name breaks the sync